pos_cashbox/tests/test_default.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+import odoo.tests
+
+
+@odoo.tests.common.at_install(True)
+@odoo.tests.common.post_install(True)
+class TestUi(odoo.tests.HttpCase):
+
+    def test_01_cashbox_is_opened(self):
+        self.phantom_js(
+            '/web?debug=assets',
+
+            "odoo.__DEBUG__.services['web_tour.tour']"
+            ".run('pos_cashbox_tour')",
```

Comment:
The is not covered your js at all. It opens POS, while your js operates in backend.

Make a tour that

1. opens the widget, clicks the button and gets warning that proxy ip is not specifed
2. via mock js it gets fake response and user sees "wow"

Reply:
@yelizariev We can open a new problem for creating the test